addon/ng2/blueprints/ng2/files/__path__/index.html

```diff
@@ -22,9 +21,15 @@
   <link rel="apple-touch-icon" sizes="180x180" href="/icons/apple-touch-icon-180x180.png">
   <link rel="apple-touch-startup-image" href="/icons/apple-touch-icon-180x180.png">
   <% } %>
-  
+  <% if (!arguments[0].files) { 
```

Comment:
What is the purpose of this templating logic? Webpack entry points together with the `HtmlWebpackPlugin` should take care of inserting the css files here.